src/Uno.UWP/ApplicationModel/Contacts/ContactManager.Android.cs

```diff
@@ -0,0 +1,100 @@
+﻿#if __ANDROID__
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Foundation;
+
+namespace Windows.ApplicationModel.Contacts
+{
+	public partial class ContactManager
+	{
+
+		public static IAsyncOperation<ContactStore> RequestStoreAsync() => RequestStoreAsync(ContactStoreAccessType.AllContactsReadOnly);
+
+		public static IAsyncOperation<ContactStore> RequestStoreAsync(ContactStoreAccessType accessType) => RequestStoreAsyncTask(accessType).AsAsyncOperation<ContactStore>();
+
+		private static async Task<ContactStore> RequestStoreAsyncTask(ContactStoreAccessType accessType)
+		{
+			// UWP: AppContactsReadWrite, AllContactsReadOnly, AllContactsReadWrite (cannot be used without special provisioning by Microsoft)
+			// Android: Manifest has READ_CONTACTS and WRITE_CONTACTS, no difference between app/limited/full
+			// using only AllContactsReadOnly, other: throw NotImplementedException
+
+			if (accessType != ContactStoreAccessType.AllContactsReadOnly)
+				throw new NotImplementedException();

```

Comment:
Do not throw not implemented in this case. It is implemented, it's an `InvalidOperationException` maybe, or something else.

Reply:
Why not NotImplementedException()? It is unimplemented on this platform.
AppContactsReadWrite has no meaning in Android API, and AllContactsReadWrite is simply NotImplementedException() - but it can be implemented. Just I doesn't use 'contact write' in my app, so I didn't implement such functionality.

Reply:
I undertand. Will it be possible using the android APIs at some point? If not, then maybe `NotSupportedException` is better suited.

Reply:
Yes, it can be implemented. I just have not enough time.
I'm trying to implement call log and sms store (contacts is side effect of converting "SMS backup" app from UWP to Uno). When I would add these two (calllog, sms), I will have nine open PR :) [10, counting previous geolocator, waiting for help with system asking for permission - same code as commented-out here, in ContactManager.RequestStoreAsyncTask

And I'm trying to learn much more from Git... :)

Reply:
@pkar70 Read this book: https://git-scm.com/book/en/v2 That's by far the best way to learn git.